core-models/src/main/scala/com/mesosphere/usi/core/models/PodSpec.scala

```diff
@@ -0,0 +1,3 @@
+package com.mesosphere.usi.core.models
```

Comment:
Wdyt about putting:
- `PodSpec` into `core.models.framework`
- `PodRecord` and `PodReservation` into `core.models.scheduler` 
to make it more obvious that they are used on different levels?

Reply:
Discussed offline, no longer needed.